UIkitDZ/VC/LeftViewController.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  ViewController.swift
+//  UIkitDZ
+//
+//  Created by Timofey Privalov on 30.06.2021.
+//
+
+import UIKit
+
+final class LeftViewController: UIViewController {
```

Comment:
```suggestion
final class LeftViewController: UIViewController {
```
название контролера не должны быть первый второй или лево право